Makefile

```diff
@@ -4,9 +4,9 @@ PHP-CS-FIXER=./php-cs-fixer-v2.phar
 PHPUNIT=vendor/bin/phpunit
 
 DOCKER_RUN=docker run -t --rm -v "$$PWD":/opt/infection -w /opt/infection
-DOCKER_RUN_70=flock devTools/*php70*.json $(DOCKER_RUN) infection_php70
-DOCKER_RUN_71=flock devTools/*php71*.json $(DOCKER_RUN) infection_php71
-DOCKER_RUN_72=flock devTools/*php72*.json $(DOCKER_RUN) infection_php72
+DOCKER_RUN_70=type -t flock && flock devTools/*php70*.json & $(DOCKER_RUN) infection_php70
+DOCKER_RUN_71=type -t flock && flock devTools/*php71*.json & $(DOCKER_RUN) infection_php71
+DOCKER_RUN_72=type -t flock && flock devTools/*php72*.json & $(DOCKER_RUN) infection_php72
```

Comment:
This single ampersand here makes `flock` useless in this command. In other words, Docker will always run no matter if `flock` is found or not.

Reply:
> Docker will always run no matter if flock is found or not

that was exactly what I wanted to achieve